Makefile

```diff
@@ -18,12 +19,12 @@ dependencies:
 	luarocks make --local rockspec
 
 build:
-	$(DOCKER_COMPOSE) build
+	$(S2I) build . openresty-builder docker-gateway-test
```

Comment:
should we download the openresty-builder and build it locally or depend on the quay.io/3scale/openresty-builder:SOME_VERSION ? 

I would go for the second option.